XcodeServerSDK/Utils/HTTPUtils.swift

```diff
@@ -30,15 +30,13 @@ public class HTTP {
             //try to cast into HTTP response
             if let httpResponse = response as? NSHTTPURLResponse {
                 
-                if error != nil {
-                    
+                guard error != nil else {
```

Comment:
Are you sure you're using `guard` correctly? AFAIK you want to guard for something being true, in which case you want the control flow to continue. Otherwise, you want it to go into the scope.

In this case, the old code goes into the `//error in the networking stack` block iff `error != nil`. However, with your change with `guard`, now the block of code will be executed if `error == nil`, which is the opposite.

I think, I haven't used `guard` myself, but this is how I remember they work.